broker/utils.go

```diff
@@ -25,10 +25,6 @@ func findDeployment(c compose.Client, name string) (*composeapi.Deployment, erro
 	return nil, fmt.Errorf("deployment: not found")
 }
 
-func JDBCURI(scheme, hostname, port, dbname, username, password string) string {
```

Comment:
We can work that out when we come to it. I think we'll need to refactor and extract out how we construct the credentials anyway once we have more than one type of service to cater for.